package/fuseki/package

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright (c) 2025 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(fuseki)
```

Comment:
This should be split into two packages, fuseki and gnugo. fuseki would depend on the gnugo package. You can have both packages defined in this recipe, or split them into two recipes.

Reply:
I'll do them as two recipes. Do you want one PR for both, or shall I submit gnugo one first separately?

Reply:
A single PR is fine.